products/views.py

```diff
@@ -1,3 +1,63 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+from products.models  import Product
+from users.utils      import login_decorator
+
+class ProductDetailView(View):
+    @login_decorator
+    def post(self, request, id):
```

Comment:
```suggestion
    def get(self, request, id):
```
상세 데이터 조회 이므로 get method를 사용해 주세요.

Reply:
get 메소드로 변경하여 진행했습니다.